### PR TITLE
Updated development guide

### PR DESCRIPTION
**Pull request by kariharju**

### CODE REVIEW

File: docs/develop.md [hunk 1]
```diff
@@ -2,22 +2,28 @@
 Developing
 -----------
 
-Install NodeJs (https://nodejs.org/en/) -- make sure that `node` and `npm` are in the `PATH`.
+🚀 Base installations and first run
 
-Install Yarn (https://yarnpkg.com/) -- make sure that `yarn` is in the `PATH`.
+Install [NVM](https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating) to get the different Node versions easily 
+- `nvm install 20.12.2` installs correct Node version
```
[the remark this answers is not in the document]
Comment (kariharju) [reply docs/develop.md:8]: Not opposed, but never used these.. what's the format?

[the remark this answers is not in the document]
Comment (custompointofview) [reply docs/develop.md:8]: It's just this: 
```
v21.7.3
```

**Review by custompointofview — COMMENTED** [review]
Should the `.vsix` creation be added as well?

File: docs/develop.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kariharju) [reply docs/develop.md:8]: Added the .nvmrc file but I need to get this in so testing that I'll leave for a later date.

**Pull request merged**